src/gui/mainwindow.cpp

```diff
@@ -1481,7 +1481,8 @@ void MainWindow::on_actionSearchWidget_triggered()
 
         // Check if python is already in PATH
         if (pythonVersion > 0)
-            Logger::instance()->addMessage(tr("Python found in %1").arg("PATH"), Log::INFO); // Prevent translators from messing with PATH
+            // Prevent translators from messing with PATH
+            Logger::instance()->addMessage(tr("Python found in %1 %2").arg("PATH").arg(qgetenv("PATH").constData()), Log::INFO);
```

Comment:
You do realize that path might look like "/usr/local/bin:/usr/bin:/etc/bin", right? This won't be helpful for users. It won't tell them exactly where we found it. But anyway, you can leave it there with a slight change:
```c++
Logger::instance()->addMessage(tr("Python found in %1: %2", "Python found in PATH: /usr/local/bin:/usr/bin:/etc/bin").arg("PATH").arg(qgetenv("PATH").constData()), Log::INFO);
```